doc/lightning-listfunds.7.md

```diff
@@ -37,7 +37,7 @@ On success, an object is returned, containing:
     - **reserved_to_block** (u32): Block height where reservation will expire
 - **channels** (array of objects):
   - **peer_id** (pubkey): the peer with which the channel is opened
-  - **our_amount_msat** (msat): available satoshis on our node’s end of the channel
+  - **our_amount_msat** (msat): available satoshis on our node's end of the channel
```

Comment:
nit, this change should be done in https://github.com/ElementsProject/lightning/blob/a063d5dd417b0f254718cb9ce1a1f0fc2e039155/doc/schemas/listfunds.schema.json#L152

In this part of the schema

```
          "our_amount_msat": {
            "type": "msat",
            "description": "available satoshis on our node’s end of the channel"
          },
```

Reply:
Oops. I pushed a fix for this. Thanks!